src/Rules.Framework/Evaluation/OperatorsMetadata.cs

```diff
@@ -0,0 +1,153 @@
+namespace Rules.Framework.Evaluation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.Linq;
+    using Rules.Framework.Core;
+
+    internal static class OperatorsMetadata
+    {
+        private static readonly IDictionary<Operators, OperatorMetadata> allOperatorsMetadata;
+
+        private static readonly IDictionary<string, OperatorMetadata> allOperatorsMetadataBySupportedCombination;
+
+        private static readonly IEnumerable<OperatorMetadata> operatorsMetadata = new[]
+                        {
```

Comment:
May you please fix the extra spacing for {

Reply:
Fixed, thank you Carlos :)